scripts/RT/magic.R

```diff
@@ -26,6 +29,7 @@ if (length(args) == 0) {
   weppgraphfile=args[2];
   weppsoilfile=args[3];
   Year=as.numeric(args[4]);
+  weppmgmt=args[5]
   DayofYear=as.numeric(args[5]);

```

Comment:
This looks like a typo with args[5] reused with `DayofYear`

Reply:
I think the "weppmgmt=args[5]" can be changed to "...args[7]" without causing anything to implode